MAINTAINERS

```diff
@@ -403,6 +403,8 @@ made through a pull request.
 				"fredlf",
 				"james",
 				"sven",
+				"spf13",
+				"mary"
```

Comment:
Should this be ordered alphabetically as well? (no sure, and sorry for being pedantic)